### DISCUSSION TITLE
Email Verification --  Cannot GET [Aug 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3391] [Discord Threads]
> 🙋‍♂️ **𝘿𝙚𝙛𝙖𝙘𝙚𝙙𝘿𝙖𝙩𝙖**  
## Anyone have any ideas on email verification? I see it in the mySQL DB under `nc_users_v2 1` and `email_verification_token` but it's like I cant talk to that.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1009992319698751528/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/1009992320034287687/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **𝘿𝙚𝙛𝙖𝙘𝙚𝙙𝘿𝙖𝙩𝙖**  
#### Would it have anything todo with `NC_BACKEND_URL` or `NC_PUBLIC_URL` ?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1009992321569398884)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
